hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/ScalaCollector.java

```diff
@@ -20,27 +20,35 @@
 
 import org.apache.flink.util.Collector;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * A mock {@link Collector} that used in  {@link TestFunctionWrapper}.
  */
 class ScalaCollector<T> implements Collector<T> {
```

Comment:
The class name implies that the result should be singleton.

Reply:
`Scalar`? ok, then we should rename it if multiple emitted records is expected.